compiler/core/src/core/config.re

```diff
@@ -194,4 +227,26 @@ let load =
            })
          )
     )
-  };
\ No newline at end of file
+  };
+
+let toJson = (compilerVersion: string, config: t): Js.Json.t =>
+  Json.(
+    Encode.object_([
+      ("version", Encode.string(compilerVersion)),
+      (
+        "paths",
+        Encode.object_([
+          ("workspace", Encode.string(config.workspacePath)),
+          ("colors", Encode.string(config.colorsFile.path)),
+          ("textStyles", Encode.string(config.textStylesFile.path)),
+          ("shadows", Encode.string(config.shadowsFile.path)),
```

Comment:
isn't there `gradients` as well?

Reply:
Yeah. We don't generate any code for it yet though. I could put the path in if we wanted.

Reply:
Ah got. Nevermind then